.github/pull_request_template.md

```diff
@@ -75,3 +75,12 @@ Operator (reindex)
 Operator
 
 - [ ] Unassigned PR
+
+---  
+
+Operator (prod)
+- [ ] Revert `prod` hotfixes for this PR                    <sub>or there are no hotfixes for this PR in `prod`</sub>
+- [ ] Merge `develop` into `prod`
```

Comment:
I've not done a `prod` release; but do we also do a `prod` -> `dev` merge as well? Or am I imaging that I heard that in a discussion between @hannes-ucsc  and @noah-aviel-dove for the last hot-fix?


Also see:
1280674d4197ca5746895a0eeeee43f6b69a63a5

Reply:
Good questions. I'll bring this up in PL.